### ISSUE TITLE
Any way to use OpenBubbles on web?

### ISSUE THREAD

**Issue by andretechguy** [TaeHagen/bluebubbles-app#55]
Hi, since OpenBubbles is based on BlueBubbles, is there any way you can use it on web? Thanks

**Comment by TaeHagen** [comment]
No, there is not. Web does not allow direct socket connections we require to talk with Apple Push Services.
